spec/pricing/nsm_spec.rb

```diff
@@ -362,16 +506,27 @@
                 assessed_total_inc_vat: 72.86,
                 at_least_one_claimed_work_item_assessed_as_type_with_different_summary_group: false,
               },
+              additional_fees: {
```

Comment:
I've got at this in other comments, but basically this section is in the wrong place. If we want an `additional fees` bit, it shouldn't be inside the cost summary object, it should be at the same level as the `letters_and_calls` bit. That would also allow us to remove the utterly irrelevant `at_least_one_claimed_work_item_assessed_as_type_with_different_summary_group` attribute.